src/navigation/stacks/ProfileStacks/CreateProfileStack.tsx

```diff
@@ -0,0 +1,167 @@
+import { useState } from 'react';
+import { Ionicons } from '@expo/vector-icons';
+import CountryPicker, { Country } from 'react-native-country-picker-modal';
+import {
+  Image,
+  StyleSheet,
+  TouchableOpacity,
+  Platform,
+  ScrollView,
+  KeyboardAvoidingView,
+} from 'react-native';
+import { Button, Input, Text, View } from 'src/components';
+import { ScreenTemplate } from 'src/templates';
+
+export default function CreateProfileStack() {
+  const genderOptions = ['Male', 'Female'];
+  const [fullname, setFullname] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+  const [country, setCountry] = useState<Country | null>(null);
+  const [gender, setGender] = useState('');
+
+  const handleSubmit = () => {
+    console.log('created account with values: ', fullname, phoneNumber, gender, country?.name);
+  };
+
+  return (
+    <ScreenTemplate>
+      <ScrollView contentContainerStyle={styles.scrollContainer}>
+        <View style={styles.header}>
+          <View style={styles.profileImageWrapper}>
+            <Image
+              source={{
+                uri: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRYR2tzlAo8Mr4r_1MNOOVAsW-NXHnGmCuQwGTfda_x9RBu6YWP6X1ohBGSVtIHo4rzKjw&usqp=CAU',
+              }}
+              style={styles.profileImage}
+            />
+            <TouchableOpacity style={{ ...styles.addPhotoButton }}>
+              <Ionicons name="add" size={24} />
+              <Text>add photo</Text>
+            </TouchableOpacity>
+          </View>
+        </View>
+        <KeyboardAvoidingView
+          behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+          style={{ gap: 10 }}
+        >
+          <View>
+            <Text style={{ fontSize: 24, fontWeight: 'bold' }}>Profile</Text>
```

Comment:
All Styles should be in a separate file (no matter how small ) and  used in actual component file.